HeatStroke/Transform/HierarchicalTransform.h

```diff
@@ -77,6 +77,11 @@ namespace HeatStroke
 		virtual std::vector<HierarchicalTransform*>::const_iterator begin() const	{ return m_vChildren.begin(); }
 		virtual std::vector<HierarchicalTransform*>::const_iterator end() const		{ return m_vChildren.end(); }
 
+		void ManuallyUpdateParent() { 
```

Comment:
multi-line functions shouldn't be in the header file.  Move this to the source file and fix the bracing to be consistent with the rest of the code.